src/store/modules/game.js

```diff
@@ -328,24 +334,24 @@ export default {
       });
     },
 
-   async requestSpectate(context, gameId) {
-    return new Promise((resolve, reject) => {
-      io.socket.get(
-        '/game/spectate',
-        {
-          gameId,
-        },
-        function handleResponse(res, jwres) {
-          if (jwres.statusCode === 200) {
-            context.commit('updateGame', res);
-            context.commit('setMyPNum', 0);
-            return resolve();
-          }
-          return reject(new Error('Unable to spectate game'));
-        },
-      );
-    });
-   },
+    async requestSpectate(context, gameId) {
+      return new Promise((resolve, reject) => {
+        io.socket.get(
+          '/game/spectate',
+          {
+            gameId,
+          },
+          function handleResponse(res, jwres) {
+            if (jwres.statusCode === 200) {
+              context.commit('updateGame', res);
+              context.commit('setMyPNum', 0);
+              return resolve();
+            }
+            return reject(new Error('Unable to spectate game'));
+          },
+        );
+      });
+    },
```

Comment:
It looks like this is overly indented past where the other functions in this file are